README.md

```diff
@@ -4,103 +4,99 @@ This package provides a set of language helpers, utility functions, and classes
 It includes APIs for feature detection, asynchronous and streaming operations, basic event handling,
 and making HTTP requests.
 
-## Features
+## Installation
 
-### Feature Detection
+This package is currently in Alpha with a initial stable release scheduled for later this year. You can install the Alpha
+through npm:
```

Comment:
The dojo2 package currently available through npm is not this package. Cloning the repo is currently the only way to go.